lib/client.js

```diff
@@ -143,6 +155,30 @@ extend(Raven.prototype, {
     return this;
   },
 
+  makeErrorHandler: function () {
+    var self = this;
+    var calledUncaughtHandler = false;
+    var calledFatalError = false;
+    var firstError;
+    return function (err) {
+      if (!calledUncaughtHandler) {
+        firstError = err;
+        calledUncaughtHandler = true;
```

Comment:
I did consider this, but it's possible to literally `throw false` or `throw undefined`:
```javascript
process.on('uncaughtException', function (err) {
  console.log('uncaught: ', typeof err);
});
throw undefined;
```
outputs:
```
uncaught: undefined
```
😅